Standards/scs-0106-v1-evaluate-current-state-of-cloudmon.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Evaluating the deployment of CloudMon in SCS infrastructure
+type: Decision Record
+---
+
+## Introduction
+
+In the fast-paced environment of modern cloud computing, ensuring the reliability and performance
+of cloud infrastructure is paramount for organizations. Effective monitoring and testing framework
+are essential tools in this endeavor, enabling proactive identification and resolution of issues before
```

Comment:
e.g. most monitoring doesn't really allow proactive resolution of issues.

Monitoring is just analysing live data after the fact. e.g. you can see an API is slow in monitoring, or you can see a disk filling up. But all the activities monitoring allows a DevOps Team to do are _reactive_. You notice a problem in monitoring, then you react.

A testing framework, on the other hand, _can_ of course be used to detect problems e.g. before deployment.